readme.txt

```diff
@@ -20,9 +20,9 @@ A wordpress plugin to automatically resize and improve upon wysiwyg and textarea
 ## Usage
 - Install and activate like any other wordpress plugin
 - `[data-wysiwyg-slug]` is added to *every* wysiwyg field!
-- Add `autosize` as a class to a WYSIWYG or Textarea-Field to enable autosize for this field
+- plugin ist enabled by default on all textareas and wysiwyg fields. You can disable it by adding the class `no-autosize` to a field.
+- You can set the enabled default to false via the `acf-autosize/enabledByDefault`-filter. After that you can reenable single fields by adding the class `autosize`.
```

Comment:
same as above. In the long run it would be interesting to generate the `readme.txt` automatically ⭐️

Reply:
absolutely!